components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml

```diff
@@ -393,6 +396,24 @@ components:
           $ref: '#/components/schemas/AuthenticatorListResponse'
         cors:
           $ref: '#/components/schemas/CORSConfig'
+    RealmConfig:
+      type: object
+      properties:
+        adminUser:
+            type: string
+            description: "Username of the super admin user in the system."
+            example: "admin"
+            readOnly: true
+        adminRole:
+            type: string
+            description: "Name of the super admin role in the system."
+            example: "Internal/admin"
+            readOnly: true
+        everyoneRole:
+            type: string
+            description: "Name of the everyone role in the system."
+            example: "Internal/everyone"
```

Comment:
is this everyone or Internal/everyone ?

Reply:
`Internal/everyone`. This is the value returned by the osgi service.